README-CN.md

```diff
@@ -69,12 +71,12 @@ scrape_configs:
 ```shell
 $ docker run -d --restart=always --name nebula-stats-exporter -p 9100:9100 \
     -v "<<PATH_OF_CONFIG_FILE>>:/config.yaml" \
-    vesoft/nebula-stats-exporter:v0.0.6 --bare-metal --bare-metal-config=/config.yaml
+    vesoft/nebula-stats-exporter:v3.1.0 --bare-metal --bare-metal-config=/config.yaml
```

Comment:
use kernel release version?

Reply:
yes, because release-tag will use v3.1.0, I think unite is better

Reply:
Is the version definition strictly consistent, or just the first two bits? 

For examples, 

1. nebula release v3.1.0
2. stats release v3.1.0
3. you find bug in stats, then stats release v3.1.1
4. nebula release v3.1.1

So what version is the stats?

Reply:
if stats don't have change, stats release v3.1.1, otherwise v3.1.2, promise x.y compatible nebula